### ISSUE TITLE
How do I assign different projects to different "admin"?

### ISSUE THREAD

**Issue by NielsenSPereira** [opencv/cvat#5617]
### My actions before raising this issue
- [x ] Read/searched [the docs](https://github.com/cvat-ai/cvat/tree/master#documentation)
- [ ] Searched [past issues](/issues)

 I'd like that each "admin" can see each spcefic Project and not all the Projects.

### Expected Behaviour
 Each "admin" only see the Projects that were assigned to him

### Current Behaviour
 every "admin" can see all the Projects

### Possible Solution
 Assigning "admin" to Project

### Steps to Reproduce (for bugs)

1.
1.
1.
1.

### Context
 For security reason, it's not expected that every "admin" can see all the Projects

### Your Environment

- Git hash commit (`git log -1`):
- Docker version `docker version` (e.g. Docker 17.0.05): Docker 20.10.20
- Are you using Docker Swarm or Kubernetes? Docker Swarm ?
- Operating System and version (e.g. Linux, Windows, MacOS): Ubuntu 18.04
- Code example or link to GitHub repo or gist to reproduce problem:
- Other diagnostic information / logs:
    <details>
    <summary>Logs from `cvat` container</summary>
    </details>

**Comment by bsekachev** [comment]
Hi, What do you mean under ``admin``? 

Usually it is a superuser who can see everything. If you want users do not see all the projects, do not grant them admin rights

**Comment by bsekachev** [comment]
@NielsenSPereira 

If John is admin there is no way to restrict visibility anyhow for him. Admin is a person who is responsible for the whole CVAT instance. 

Are you sure you want to assign users to admin role? What is your use-case?

**Comment by NielsenSPereira** [comment]
@bsekachev 
I understand that, what I think I need is a Spervisor permisson.
I have one Project with 12 tasks and one person (Supervisor A), will supervise  the annotation of the 6 users (each one has 2 tasks), so I assign him as "admin" Group for viewing all 12 tasks;
But I have another Project with 4 tasks and 4 different users and a different Supervisor ((Supervisor B)!
So, how can I assign this "Supervisor" permission to each supervisor?

Or assign him "Supervisor A" as "user" in all 12 Tasks; but when I click on Assignee + (Add user) it opens a pop-up to create another user and didn't show the list of current users, so I am a bit confuse to use this second option.

**Comment by bsekachev** [comment]
@NielsenSPereira 

Just assign "Supervisor A" to the first project. The supervisor will see all the task only inside the project. 
And assign "Supervisor B" to the second project. 

Do not need to assign "admin" role for them.

**Comment by bsekachev** [comment]
> it opens a pop-up to create another user and didn't show the list of current users, so I am a bit confuse to use this second option.

I am not sure I understand you here. When you click it opens pop-up where you need to choose existing user, there is no suggestion to create them:

<img width="861" alt="image" src="https://user-images.githubusercontent.com/40690378/214113439-d3072034-2f2f-4d7b-9f4f-e1b9bcb021cf.png">

**Comment by NielsenSPereira** [comment]
I am assigning in the Admin Page. 
Can I assign a Task to more than one "user"? 
When I click on the Add (+) Assignee, it opens the popup window, but we can't see the list of users; it seems that we have to create a new one!
![Assignee](https://user-images.githubusercontent.com/55050470/214116672-3126c46f-6ab2-4dec-9d0a-6c65c03182fa.jpg)

![Assignee_add](https://user-images.githubusercontent.com/55050470/214116726-5ea2a092-921c-4954-8edb-68a58690d3f9.jpg)

**Comment by NielsenSPereira** [comment]
I think that this Add button (+) should enable to add an Assignee (from the user list) and not a New User.

**Comment by bsekachev** [comment]
Please, use the proposed way to assign users.

> Can I assign a Task to more than one "user"?

No, but you can assign different users to different jobs of the same task. These users will be able to annotate the task in parallel this way. 
If you want to more people have access to shared resources, consider using [organizations](https://opencv.github.io/cvat/docs/manual/advanced/organization/)

**Comment by bsekachev** [comment]
Not sure what happens on your side, if it is still relevant please open another issue. I will close this one.
